ios/RNCWebViewManager.m

```diff
@@ -128,6 +128,10 @@ - (UIView *)view
   view.keyboardDisplayRequiresUserAction = json == nil ? true : [RCTConvert BOOL: json];
 }
 
+RCT_CUSTOM_VIEW_PROPERTY(allowFileAccessFromFileURLs, BOOL, RNCWebView) {
+    view.allowFileAccessFromFileURLs = json == nil ? false : [RCTConvert BOOL: json];
```

Comment:
This appears to be a private API and could cause apps to be rejected.

https://stackoverflow.com/a/44365081/204044

Any thoughts on that?

Reply:
I’ve used this in the past with production applications, and I’ve not had an issue with apps being rejected. 

However - you’re absolutely right, and I hadn’t considered that when I put up the PR. If this isn’t a risk we want to introduce to the project, I completely understand and we can close this out.